src/client.ts

```diff
@@ -4,30 +4,31 @@ import type { NGIOResponse } from "./types/objectModels";
 
 export type NewgroundsConfig = {
     debug?: boolean;
+    soundProxy?: string;
 };
 
 export class NewgroundsClient {
     private _appID: string;
     private _cipher: string;
     private _sessionID: string | null;
     private _debug: boolean;
+    public soundProxy: string;
 
     constructor(appID: string, cipher: string, config?: NewgroundsConfig) {
         this._appID = appID;
         this._cipher = cipher;
         this._debug = config?.debug ?? false;
+        this.soundProxy =
+            config?.soundProxy ?? "https://cors.niceeli.workers.dev/?";
 
         const url = new URL(globalThis.location.href);
         this._sessionID = url.searchParams.get("ngio_session_id") ?? null;
 
-        if (!this._sessionID) {
-            const startSession = async () => {
+        !this._sessionID &&
+            (async function startSession() {
                 const sessionStarted = await this.call("App.startSession");
                 this._sessionID = sessionStarted.result.data.session.id;
-            };
-
-            startSession();
-        }
+            })();
```

Comment:
The same about function shortening

Reply:
fixed in [8f7666f](https://github.com/lajbel/newgrounds-js/pull/3/commits/8f7666f7d576984ac3c82a7e0b50a1d7102c9a24)